hudi-flink/src/main/java/org/apache/hudi/HoodieFlinkStreamer.java

```diff
@@ -160,6 +156,19 @@ public static void main(String[] args) throws Exception {
         + "(using the CLI parameter \"--props\") can also be passed command line using this parameter.")
     public List<String> configs = new ArrayList<>();
 
+    @Parameter(names = {"--record-key-field"}, description = "Record key field. Value to be used as the `recordKey` component of `HoodieKey`.\n"
+        + "Actual value will be obtained by invoking .toString() on the field value. Nested fields can be specified using "
+        + "the dot notation eg: `a.b.c`. By default `uuid`.")
+    public String recordKeyField = "uuid";
+
+    @Parameter(names = {"--partition-path-field"}, description = "Partition path field. Value to be used at \n"
+        + "the `partitionPath` component of `HoodieKey`. Actual value obtained by invoking .toString(). By default `partitionpath`.")
+    public String partitionPathField = "partitionpath";
+
+    @Parameter(names = {"--partition-path-field"}, description = "Key generator class, that implements will extract the key out of incoming record.\n"
```

Comment:
little mistake   `--partition-path-field`

Reply:
Oops, thanks for the reminder.